src/components/browse/components/VirtualHrefController.js

```diff
@@ -194,10 +194,6 @@ export class VirtualHrefController extends React.PureComponent {
                     throw new Error("Did not get back a search response, request was potentially aborted.");
                 }
 
-                if (typeof globalNavigate.updateUserInfo === "function") {
-                    globalNavigate.updateUserInfo();
-                }
```

Comment:
No longer needed here since is called after every AJAX request anyways via AJAXSettings.onLoad